<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserInterfaceSettings</class>
 <widget class="QWidget" name="UserInterfaceSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>678</width>
    <height>972</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="margin">
    <number>6</number>
   </property>
   <item>
    <widget class="VerticalOnlyScroller" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>662</width>
        <height>956</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="chatGroupBox">
         <property name="title">
          <string>Chat</string>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <property name="margin">
           <number>9</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Base font:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QFontComboBox" name="txtChatFont">
              <property name="editable">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="txtChatFontSize">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="suffix">
               <string>px</string>
              </property>
              <property name="prefix">
               <string>Size: </string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="textStyleLabel">
            <property name="toolTip">
             <string>New text styling preference may not load until qTox restarts.</string>
            </property>
            <property name="text">
             <string>Text Style format:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="textStyleComboBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Select text styling preference.</string>
            </property>
            <property name="currentIndex">
             <number>1</number>
            </property>
            <item>
             <property name="text">
              <string>Plaintext</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Show formatting characters</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Don't show formatting characters</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QCheckBox" name="useNameColors">
            <property name="text">
             <string>Use colored nicknames in group chats</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="newMessageGroup">
         <property name="title">
          <string>New message</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <property name="topMargin">
           <number>6</number>
          </property>
          <item>
           <widget class="QCheckBox" name="notify">
            <property name="toolTip">
             <string comment="tooltip for Notify setting">Show a notification when you receive a new message and the window is not selected.</string>
            </property>
            <property name="text">
             <string>Notify</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_11">
            <property name="leftMargin">
             <number>40</number>
            </property>
            <item>
             <widget class="QCheckBox" name="groupOnlyNotfiyWhenMentioned">
              <property name="toolTip">
               <string comment="toolTip for Group chats only notify when mentioned">Only notify about new messages in group chats when mentioned.</string>
              </property>
              <property name="text">
               <string>Group chats only notify when mentioned</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="notifySound">
              <property name="text">
               <string>Play sound</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <property name="leftMargin">
               <number>40</number>
              </property>
              <item>
               <widget class="QCheckBox" name="busySound">
                <property name="text">
                 <string>Play sound while Busy</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QCheckBox" name="desktopNotify">
              <property name="text">
               <string>Notify via desktop notifications</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="notifyHide">
              <property name="text">
               <string>Hide message sender and contents</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="showWindow">
            <property name="toolTip">
             <string comment="tooltip for Show window setting">Open qTox's window when you receive a new message and no window is open yet.</string>
            </property>
            <property name="text">
             <string>Open window</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="contactListGroup">
         <property name="title">
          <string>Contact list</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QCheckBox" name="cbGroupchatPosition">
            <property name="toolTip">
             <string comment="toolTip for groupchat positioning">If checked, group chats will be placed at the top of the friends list, otherwise, they'll be placed below online friends.</string>
            </property>
            <property name="text">
             <string>Place groupchats at top of friend list</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbCompactLayout">
            <property name="toolTip">
             <string comment="toolTip for compact layout setting">Your contact list will be shown in compact mode.</string>
            </property>
            <property name="text">
             <string>Compact contact list</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cbSeparateWindow">
            <property name="text">
             <string>Multiple windows mode</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_12">
            <property name="leftMargin">
             <number>40</number>
            </property>
            <item>
             <widget class="QCheckBox" name="cbDontGroupWindows">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Open each chat in an individual window</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="cbShowIdenticons">
            <property name="toolTip">
             <string comment="toolTip for show identicons">If enabled, every contact without an avatar will have a generated icon based on their Tox ID instead of the default picture. Requires restart to apply.</string>
            </property>
            <property name="text">
             <string>Use identicons instead of empty avatars</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="emoticonsGroupBox">
         <property name="title">
          <string>Emoticons</string>
         </property>
         <layout class="QFormLayout" name="formLayout_4">
          <item row="0" column="0">
           <widget class="QCheckBox" name="useEmoticons">
            <property name="text">
             <string>Use emoticons</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="smileyPackLabel">
            <property name="text">
             <string extracomment="Text on smiley pack label">Smiley pack:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="smileyPackBrowser">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>80</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0,0">
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
            <item>
             <widget class="QLabel" name="smile1">
              <property name="toolTip">
               <string notr="true">:)</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="smile2">
              <property name="toolTip">
               <string notr="true">;)</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="smile3">
              <property name="toolTip">
               <string notr="true">:p</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="smile4">
              <property name="toolTip">
               <string notr="true">:O</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignTop">
             <widget class="QLabel" name="smile5">
              <property name="toolTip">
               <string notr="true">:'(</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="emoticonSizeLabel">
            <property name="text">
             <string>Emoticon size:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QSpinBox" name="emoticonSize">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="specialValueText">
             <string/>
            </property>
            <property name="suffix">
             <string> px</string>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>2147483647</number>
            </property>
            <property name="value">
             <number>25</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="themeGroupBox">
         <property name="title">
          <string>Theme</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="styleLabel">
            <property name="text">
             <string>Style:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="styleBrowser">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="themeColorLabel">
            <property name="text">
             <string>Theme color:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="themeColorCBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>118</width>
              <height>28</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QCheckBox" name="useExtraColors">
              <property name="text">
               <string>Use extra colors</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="choiceExtraColor">
              <property name="text">
               <string>Choose colors</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="timestampLabel">
            <property name="text">
             <string>Timestamp format:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QComboBox" name="timestamp">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>2</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="editable">
               <bool>true</bool>
              </property>
              <property name="insertPolicy">
               <enum>QComboBox::NoInsert</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="timeExample">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                <horstretch>2</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="text">
               <string notr="true">Time example</string>
              </property>
              <property name="textFormat">
               <enum>Qt::AutoText</enum>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="dateformatLabel">
            <property name="text">
             <string>Date format:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QComboBox" name="dateFormats">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>2</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="editable">
               <bool>true</bool>
              </property>
              <property name="insertPolicy">
               <enum>QComboBox::NoInsert</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="dateExample">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                <horstretch>2</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string notr="true">Date Example</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VerticalOnlyScroller</class>
   <extends>QScrollArea</extends>
   <header>src/widget/form/settings/verticalonlyscroller.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>scrollArea</tabstop>
  <tabstop>txtChatFont</tabstop>
  <tabstop>txtChatFontSize</tabstop>
  <tabstop>textStyleComboBox</tabstop>
  <tabstop>notify</tabstop>
  <tabstop>groupOnlyNotfiyWhenMentioned</tabstop>
  <tabstop>notifySound</tabstop>
  <tabstop>busySound</tabstop>
  <tabstop>showWindow</tabstop>
  <tabstop>cbGroupchatPosition</tabstop>
  <tabstop>cbCompactLayout</tabstop>
  <tabstop>cbSeparateWindow</tabstop>
  <tabstop>cbDontGroupWindows</tabstop>
  <tabstop>useEmoticons</tabstop>
  <tabstop>smileyPackBrowser</tabstop>
  <tabstop>emoticonSize</tabstop>
  <tabstop>styleBrowser</tabstop>
  <tabstop>themeColorCBox</tabstop>
  <tabstop>timestamp</tabstop>
  <tabstop>dateFormats</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
